nexus-writer/src/main.rs

```diff
@@ -82,6 +82,12 @@ struct Cli {
 
     #[clap(long, default_value = "127.0.0.1:9090")]
     observability_address: SocketAddr,
+
+    #[clap(long, default_value = "1048576")]
```

Comment:
How was this default selected?

Reply:
This is just 2^20, it seems like a reasonable choice from experience.